src/app/static/src/scss/style.scss

```diff
@@ -49,3 +49,10 @@ a {
 .dropdown-item:active {
   background-color: $gray-200 !important;
 }
+
+.badge-delete{
```

Comment:
`float:right` is obsolete here since you're using flex for alignment, but I think better to just go with `btn btn-danger` and no need for this class.